experimental/PropertyDDS/packages/property-shared-tree-interop/package.json

```diff
@@ -10,9 +10,19 @@
 	},
 	"license": "MIT",
 	"author": "Microsoft and contributors",
-	"main": "dist/index.js",
-	"module": "lib/index.js",
-	"types": "dist/index.d.ts",
```

Comment:
Don't we still want main/types to point customers still using node10 at our public rollup?

Reply:
I was thinking no as this package is quite new. (I think I originally removed it when I found no uses of it as a library and later saw that is its intent.) I can restore main/types.?

Reply:
I'd leave them in for customers who are integrating Fluid into a pre-existing codebase, which describes most of our 3P partners.

(...but I wouldn't hold off merging this PR for it.)

Reply:
I restored main and types when the CI build wasn't triggering.
`module` is not a supported field. When we have exports we should just remove module.